docs/defradb/guides/explain-systems.md

```diff
@@ -111,61 +115,59 @@ query @explain {
 }
 ```
 
-With the corresponding Plan Graph:
-
-Simple Explain requests are extremely fast, since it does not actually execute the constructed Plan Graph. It is intended to give transparency back to the developer, and to understand the structure and operations of how the database would resolve their request.
+Simple Explain requests are very fast since they don’t actually execute the plan graph. They’re designed to give developers a transparent view of the operations that would occur during query execution.
 
 ### Execute Explain
 
-Execute explanation differs from Simple mode because it actually executes the constructed plan graph from the request. However, it doesn't return the results, but instead collects various metrics and runtime information about how the request was executed, and returns it using using the same rendered plan graph structure that the Simple Explain does. This is similar to EXPLAIN ANALYZE from PostgreSQL or MySQL
+The Execute Explain request goes beyond Simple Explain by actually executing the constructed plan graph. While it doesn’t return query results, it collects various metrics and runtime information about the execution process and returns these in the same plan graph format used by the Simple Explain request. This functionality is akin to the EXPLAIN ANALYZE feature in databases like PostgreSQL or MySQL.
 
-You can create an Execute Explain by specifying the explain type using the directive type​arguments @explain(type: execute).
+To initiate an Execute Explain request, simply specify `@explain(type: execute)` in the query directive.
 
-The following example shows a Execute Explain request applies to an author query request.
+Here’s an example of an Execute Explain request applied to an `Author` query:
 
 ```graphql
 query @explain(type: execute) {
-	Author {
-		name
-		age
-	}
+ Author {
+  name
+  age
+ }
 }
 ```
 
 ```json
 // Response
 [
-	{
-		"explain": {
-			"executionSuccess": true,
-			"sizeOfResult":     1,
-			"planExecutions":   2,
-			"selectTopNode": {
-				"selectNode": {
-					"iterations":    2,
-					"filterMatches": 1,
-					"scanNode": {
-						"iterations":    2,
-						"docFetches":    2,
-						"filterMatches": 1
-					}
-				}
-			}
-		}
-	}
+ {
+  "explain": {
+   "executionSuccess": true,
+   "sizeOfResult":     1,
+   "planExecutions":   2,
+   "selectTopNode": {
+    "selectNode": {
+     "iterations":    2,
+     "filterMatches": 1,
+     "scanNode": {
+      "iterations":    2,
+      "docFetches":    2,
+      "filterMatches": 1
+     }
+    }
+   }
+  }
+ }
 ]
 ```
```

Comment:
todo: These code blocks no longer have consistent formatting/indenting with the rest the examples